classifier/requirements.txt

```diff
@@ -3,4 +3,4 @@ nltk
 pandas
 gensim
 python-Levenshtein
-hashlib
```

Comment:
I assume the hashlib deleted here because it is part of the python library right?